### PR TITLE
Guide to AgoraDesk

### PR DESCRIPTION
**Pull request by bitlevi**
Hello QnA! Back from vacation with a new guide.

**Here is the first draft to AgoraDesk**

Detailed step-by-step guide on how to buy and sell Bitcoin on AgoraDesk. I cover the process from account creation to trade completion, providing instructions for both buying and selling transactions. Additionally, there is the usual FAQ with answers I got directly from the telegram admin and by reading the T&S. 

### Introduction
- Dangers of KYC.
- Jargon Buster.
- Account creation instructions.

### Buying/Selling
- Buying Bitcoin, including how to filter and select offers, initiate a trade, and confirm payment.
- Selling Bitcoin, including how to deposit funds, create an advertisement, and finalize transactions.

### Learn More
- Podcast Appearances.
- FAQ.
- Useful Links.

### CODE REVIEW

File: agora.md [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+  <a href="#buying-bitcoin">Buy Sats</a> |
+  <a href="#faq">FAQ</a> |
+  <a href="#learn-more">Learn More</a> 
+</p>
+
+<p align="center">
+<img src="/assets/img/agoradesk.png" class=responsive style="max-width: 80%; height: auto;" />
+</p>
+
+## Introduction
```
Comment (BitcoinQnA) [inline agora.md:20]: Can we simplify this or change it slightly just so it's not the same as shown in the other noKYC guides?

File: agora.md [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+
+<p align="center">
+<img src="/assets/img/NKYC2.png" class=responsive width="300" height="230" maxheight="300" />
+</p>
+
+## What is AgoraDesk?
+
+AgoraDesk is an Open Source and online peer-to-peer marketplace for trading cryptocurrencies, especially designed to address the issue of KYC in Bitcoin. AgoraDesk was created by the same team that developed LocalMonero, which is an equivalent to LocalBitcoins for Monero. 
+
+With a vision to uphold the principles of privacy and freedom, AgoraDesk does not require any personal information while offering a wide range of payment methods with good liquidity on the most common. AgoraDesk is available on the clearnet, Tor network, and mobile applications on iOS and Android.
```
Comment (BitcoinQnA) [inline agora.md:40]: Final part of the first sentence doesn't read quite right. Feels unfinished?

File: agora.md [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+With a vision to uphold the principles of privacy and freedom, AgoraDesk does not require any personal information while offering a wide range of payment methods with good liquidity on the most common. AgoraDesk is available on the clearnet, Tor network, and mobile applications on iOS and Android. 
+
+## Before You Start 
+
+### Things you need to know
+
+#### Jargon Buster
+
+|----|-------|
+| [Arbitration Bond](https://learn.robosats.com/docs/robots/) | Arbitration bond is a certain amount of Bitcoin that a seller locks in the platform as a guarantee of their credibility and seriousness about a trade. |
```
Comment (BitcoinQnA) [inline agora.md:49]: All of these are linking to Robosats docs. Have you forgotten to change them?

File: agora.md [hunk 4]
```diff
@@ -0,0 +1,288 @@
...
+| [Dispute Resolution](https://learn.robosats.com/docs/robots/) | AgoraDesk provides dispute resolution if there is a disagreement between the buyer and seller. They investigate and make a decision based on evidence provided by both parties. |
+| [Fees](https://learn.robosats.com/docs/robots/) | AgoraDesk users who create advertisements are charged a 1% arbitration protection fee for every completed trade. |
+
+<br/>
+
+### Things you need to have
+
+#### Bitcoin Wallet 
+
+When purchasing non-KYC Bitcoin on Agora, you must provide a receiving Bitcoin address. You can send sats directly to your cold storage or hot wallets such as [Samourai Wallet]() or [Sparrow Wallet]() to start mixing your freshly new acquired sats!
```
Comment (BitcoinQnA) [inline agora.md:61]: Would be cool to include Envoy here!

File: agora.md [hunk 5]
```diff
@@ -0,0 +1,288 @@
...
+<img src="/assets/img/agora-buying-bitcoin-04.png" class=responsive style="max-width: 100%; height: auto;" />
+</p>
+
+To initiate a trade, enter the amount you wish to purchase, provide a new address from your wallet (avoid using the same address twice), and click on "Send Trade Request".
+
+<p align="center">
+<img src="/assets/img/agora-buying-bitcoin-03.png" class=responsive style="max-width: 100%; height: auto;" />
+</p>
+
+After sending your trade request, wait for the seller to accept your offer. Upon acceptance, you will receive the payment details via the encrypted chat function on the trade page (The chat is accessible to you and your trade partner).
```
Comment (BitcoinQnA) [inline agora.md:114]: 'The chat is accessible only to you and your trade partner' ?

File: agora.md [hunk 6]
```diff
@@ -0,0 +1,288 @@
...
+
+After making payment according to the seller's instructions, confirm by clicking "I have paid". This will notify the seller about the successful payment, ensuring the trade won't be canceled.
+
+<p align="center">
+<img src="/assets/img/agora-buying-bitcoin-05.png" class=responsive style="max-width: 100%; height: auto;" />
+</p>
+
+As the payment is confirmed, the trade settlement process starts. The status will switch to "Processing". 
+
+Now, wait for the coins to be automatically transferred to your provided wallet address. This process can take between 10 to 60 minutes, with network fees being deducted from the final amount, so you might receive slightly less than anticipated.
```
Comment (BitcoinQnA) [inline agora.md:126]: Is this bit automatic, or is it dependent on the sender confirming receipt of payment?

**Review by BitcoinQnA — COMMENTED** [review]
Excellent piece dude. Your best yet! Very impressive stuff, thank you.

Some very tiny comments/questions before I merge.

File: agora.md (hunk 1 above)
Comment (bitlevi) [reply agora.md:20]: Sure thing, I will give it a try and make this part different from other guides.

File: agora.md (hunk 3 above)
Comment (bitlevi) [reply agora.md:49]: Nice catch, I completely missed this. I copied and pasted your RoboSats grid 😆

File: agora.md (hunk 4 above)
Comment (bitlevi) [reply agora.md:61]: Sure, let's start including Envoy more often! 😎

File: agora.md (hunk 6 above)
Comment (bitlevi) [reply agora.md:126]: When the seller confirms receipt of payment, the sats are automatically released. I will reword it to make it clearer. 👍

**Comment by bitlevi** [comment]
Brilliant, thanks for reviewing this, you are a legend. I tried making it more concise for once.
I missed some easy mistakes but thankfully you have sharp eyes as usual! 🙏 

I will make these modifications in the next days and get back to you. Have a great week mate!

File: agora.md (hunk 2 above)
Comment (BitcoinQnA) [reply agora.md:40]: I've added some suggestions in bold. 

"AgoraDesk is an Open Source and online peer-to-peer marketplace for trading Bitcoin and Monero. With a vision to uphold the principles of privacy and freedom, Agora Desk does not require any personal information **and** **offers** a wide range of payment methods with good liquidity on the most common **trading pairs**. AgoraDesk was created by the same team that developed LocalMonero, which is an equivalent to LocalBitcoins for Monero."

Comment (bitlevi) [reply agora.md:40]: Sounds better! Cheers 🙏

**Comment by bitlevi** [comment]
The last pushes include:
- Modification of the introduction with updated news about dangers of KYC.
- Simplification of Agora's description.
- Addition of links for hot wallets and Envoy.
- Spelling mistakes.

Let me know if I missed anything.

**Comment by BitcoinQnA** [comment]
LOVE it! Thank you again mate
